ds/shoelace/src/vue/x-navigation-tree.vue

```diff
@@ -173,13 +174,4 @@ async function onDocumentUpdate(page: PageData) {
   </sl-tree>
 </template>
 
-<style scoped>
-:deep(a) {
-  text-decoration: none;
-  color: var(--cr-base-text-color);
-}
-/* Disable hand cursor on items, since we disable the default click action. */
-:deep(sl-tree-item)::part(base) {
-  cursor: default;
-}
-</style>
+<style scoped></style>
```

Comment:
Can simply be removed since empty.
```suggestion
<style scoped></style>
```